addons/purchase/data/mail_template_data.xml

```diff
@@ -60,7 +60,7 @@
             <field name="report_template" ref="action_report_purchase_order"/>
             <field name="report_name">PO_${(object.name or '').replace('/','_')}</field>
             <field name="lang">${object.partner_id.lang}</field>
-            <field name="user_signature" eval="False"/>
+
```

Comment:
You add a lot of blank lines in both the community and the enterprise PRs.

Can you please remove them?